### ISSUE TITLE
hwcomposer broken on wlroots

### ISSUE THREAD

**Issue by mochaaP** [waydroid/android_hardware_waydroid#53]
`hwcomposer` stopped working on sway (and other wlroots based compositors, e.g. cage) since build 20230923.

The last known good build is 20230916.

<https://github.com/waydroid/android_hardware_waydroid/compare/ebb2c3e...lineage-18.1>

Relevant logs:

```log
Control message: Could not find 'android.hardware.graphics.composer@2.1::IComposer/default' for ctl.interface_start from pid: 23 (/system/bin/hwservicemanager)
Unable to set property "ctl.interface_start" to "android.hardware.graphics.composer@2.1::IComposer/default": error code: 0x20
```

To reproduce: `cage waydroid session start`

---

Environment info:
```
OS: Fedora release 40 (Rawhide) x86_64
Kernel: 6.6.0-0.rc2.20230922git27bbf45eae9c.23.fc40.x86_64
WM: swayfx 0.3.2 / cage 0.1.5-1.fc40 / wlroots 0.16.2-3.fc39
CPU: Intel i7-10510U
GPU: Intel CometLake-U GT2 [UHD Graphics]
```

**Comment by aleasto** [comment]
Sorry, this is fixed now.

I have removed the builds from the OTA channel and will try to get new ones built as soon as possible

**Comment by aleasto** [comment]
`sudo waydroid upgrade` is how you'd rescue from this once the new builds are out

**Comment by mochaaP** [comment]
Whoa, this is quick. Thank you!

**Comment by mochaaP** [comment]
Confirm fixed on 20230925.
